templates/vm/guest.xml.j2

```diff
@@ -84,7 +84,12 @@
         <suspend-to-disk enabled="no" />
     </pm>
     <devices>
-        <emulator>/usr/bin/qemu-system-x86_64</emulator>
+	<emulator>/usr/bin/qemu-system-x86_64</emulator>
+	<disk type="file" device="disk">
```

Comment:
This will not work with in cluster mode.
And it will be great to have more than one disk.
You can update with something like this:
```xml
{% for disk in vm.local_disk %}
  <disk type="file" device="disk">
    <driver name="qemu" type="qcow2"/>
    <source file="/var/lib/libvirt/images/{{ disk.disk_file }}"/>
    <target dev="vda" bus="virtio"/>
  </disk>
{% endfor %}
```